<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogInputData</class>
 <widget class="QDialog" name="DialogInputData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>527</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>371</width>
    <height>527</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>371</width>
    <height>527</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sportsman Info</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Информация о спортсмене</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>271</width>
     <height>348</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Возраст (18 - 140)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="ageSpinBox">
      <property name="minimum">
       <number>18</number>
      </property>
      <property name="maximum">
       <number>140</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Вес, кг (50 - 180)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="weightSpinBox">
      <property name="minimum">
       <double>50.000000000000000</double>
      </property>
      <property name="maximum">
       <double>180.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Пол (m/f)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="sexLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Качок?</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="isJockComboBox"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Имя</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Золотых медалей (0 - 250)</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QSpinBox" name="goldMedalsSpin">
      <property name="maximum">
       <number>250</number>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Серебрянных медалей (0 - 250)</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QSpinBox" name="silverMedalsSpin">
      <property name="maximum">
       <number>250</number>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Бронзовых медалей (0 - 250)</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QSpinBox" name="bronzeMedalsSpin">
      <property name="maximum">
       <number>250</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogInputData</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogInputData</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
